### PR TITLE
First cmd to convert jpg to png

### LINKED ISSUE #2: [Feature]: Add a command to convert `jpeg` to `png`
Import `sp13/cobra` pkg and make our first command that will take in an image as `jpeg` and convert it to `png`.

### PR DESCRIPTION
**Pull request by unsuman**
Added our first command that converts a `jpeg` to `png`. This is not perfect because the app only expects a jpeg to be served and converts it to png for now!

closes #2

### CODE REVIEW

File: cmd/img.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+			return err
+		}
+		return nil
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(imgCmd)
+
+	// Here you will define your flags and configuration settings.
```
Comment (UjjwalMahar) [inline cmd/img.go:43]: For future work let's not add these comments here.

Comment (UjjwalMahar) [inline cmd/img.go:43]: For future work let's not add these comments here.

File: cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/*
+Copyright © 2024 NAME HERE <EMAIL ADDRESS>
```
Comment (UjjwalMahar) [inline cmd/root.go]: NAME HERE and EMAIL ADDRESS i needs to be discussed

File: cmd/img.go (hunk 1 above)
Comment (unsuman) [reply cmd/img.go:43]: These comments are generated by the `cobra-cli`

File: cmd/root.go (hunk 2 above)
Comment (unsuman) [reply cmd/root.go]: yup

Comment (unsuman) [reply cmd/root.go]: @UjjwalMahar we will discuss later but can you review the functionality?

**Comment by unsuman** [comment]
@UjjwalMahar Status on this PR?

**Comment by UjjwalMahar** [comment]
The code looks good need to have few modifications but that can be done in next PR I'll merge the PR

**Pull request merged**